src/Symfony/Component/DomCrawler/AbstractUriElement.php

```diff
@@ -24,7 +24,7 @@ abstract class AbstractUriElement
     protected $node;
 
     /**
-     * @var string The method to use for the element
+     * @var string|null The method to use for the element
```

Comment:
The DomCrawler form relies on this beeing nullable and the contructor accepting null for the method.

Reply:
Shouldn't it be fixed on 3.4? Same for all changed docblocks and (4.2) types declarations

Reply:
Does not seem worth it. We have alot of properties that do not declare null but actually can be null. This is just one of them related to fixing `getMethod`. But that is also more for 4.4 as it could slighly change behavior for people using AbstractUriElement alone and fixes a bug at the same time.